src/plugins/home/public/application/components/tutorial/tutorial.js

```diff
@@ -172,15 +171,39 @@ class TutorialUi extends React.Component {
     const instructionSet = this.getInstructionSets()[instructionSetIndex];
     const esHitsCheckConfig = _.get(instructionSet, `statusCheck.esHitsCheck`);
 
-    if (esHitsCheckConfig) {
-      const statusCheckState = await this.fetchEsHitsStatus(esHitsCheckConfig);
+    //Checks if a custom status check callback  was registered in the CLIENT
```

Comment:
I wish we had tests on this, but I don't think it's worth blocking this change for it. We should probably add coverage on a larger pass on this plugin.

Reply:
@joshdover I added a few unit tests that cover some scenarios. https://github.com/elastic/kibana/pull/100816/commits/5dddd48724c417914fbb3b34811dbadf1654d6ee